src/inference/src/dev/core_impl.cpp

```diff
@@ -193,33 +193,48 @@ ov::Plugin ov::CoreImpl::get_plugin(const std::string& pluginName) const {
 
         // configuring
         {
-            if (device_supports_cache_dir(plugin)) {
-                auto cacheConfig = coreConfig.get_cache_config_for_device(deviceName);
-                if (cacheConfig._cacheManager) {
-                    desc.defaultConfig[CONFIG_KEY(CACHE_DIR)] = cacheConfig._cacheDir;
+            // TODO: remove this block of code once GPU removes support of ov::cache_dir
+            // also, remove device_supports_cache_dir at all
+            {
+                OPENVINO_SUPPRESS_DEPRECATED_START
+                if (device_supports_cache_dir(plugin)) {
+                    ov::AnyMap empty_map;
+                    auto cacheConfig = coreConfig.get_cache_config_for_device(plugin, empty_map);
```

Comment:
I don't think that we need to have local variables here 
```suggestion
                    auto cacheConfig = coreConfig.get_cache_config_for_device(plugin, {});
```

Reply:
we need lvalue